deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts

```diff
@@ -0,0 +1,124 @@
+    const setBorrowSpeedCalldata = extractCalldata((await configurator.populateTransaction.setBaseTrackingBorrowSpeed(comet.address, exp(0.000578703703703703703, 15))).data);
+    const deployAndUpgradeToCalldata = extractCalldata((await proxyAdmin.populateTransaction.deployAndUpgradeTo(configurator.address, comet.address)).data);
+
+    // 4. Send some USDC from Timelock to Comet
+    // XXX assert that funds have been transferred by diffing the balances before and after
+    const sendUSDCToCometCalldata = extractCalldata((await USDC.populateTransaction.transfer(comet.address, exp(500_000, 6))).data)
+
+    // 5. Stream COMP
+    // Streaming COMP speed = (COMP/year) / (seconds in a year / avg. block time)
+    // (50 * 365) * 13.5 / 3.154e7 = 0.0078115 COMP per block
+    const comptrollerAddress = '0x3d9819210a31b4961b30ef54be2aed79b9c9cd3b';
+    const streamCOMPCalldata = ethers.utils.defaultAbiCoder.encode(['address', 'uint'], [rewards.address, exp(0.0078115, 18)]);
+
+    const txn = await deploymentManager.asyncCallWithRetry(
+      async (signer_) => (await governor.connect(signer_).propose(
+        [
+          // 1,2,3
+          configurator.address,
+          configurator.address,
+          configurator.address,
+          configurator.address,
+          configurator.address,
+          configurator.address,
+          proxyAdmin.address,
+          // 4
+          USDC.address,
+          // 5
+          comptrollerAddress
+        ],
+        [
+          // 1,2,3
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          // 4
+          0,
+          // 5
+          0,
+        ],
+        [
+          // 1,2,3
+          "updateAssetSupplyCap(address,address,uint128)",
+          "updateAssetSupplyCap(address,address,uint128)",
+          "updateAssetSupplyCap(address,address,uint128)",
+          "updateAssetSupplyCap(address,address,uint128)",
+          "updateAssetSupplyCap(address,address,uint128)",
+          "setBaseTrackingBorrowSpeed(address,uint64)",
+          "deployAndUpgradeTo(address,address)",
+          // 4
+          "transfer(address,uint256)",
+          // 5
+          "_setContributorCompSpeed(address,uint256)"
+        ],
+        [
+          // 1,2,3
+          updateCOMPSupplyCapCalldata,
+          updateWBTCSupplyCapCalldata,
+          updateWETHSupplyCapCalldata,
+          updateUNISupplyCapCalldata,
+          updateLINKSupplyCapCalldata,
+          setBorrowSpeedCalldata,
+          deployAndUpgradeToCalldata,
+          // 4
+          sendUSDCToCometCalldata,
+          // 5
+          streamCOMPCalldata,
+        ],
+        'Increase supply caps and borrow speed, seed Comet USDC reserves from Timelock, and stream COMP to CometRewards')
+      ).wait()
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    debug(`Created proposal ${proposalId}.`);
```

Comment:
Yeah I actually found a bug (fix #519) in the `ProposalConstraint` while running this locally, since I saw that 0 solutions were being run for mainnet scenarios. Going to try running scenarios again locally with the fix.